libobs/util/bmem.c

```diff
@@ -89,12 +89,18 @@ static void a_free(void *ptr)
 
 static struct base_allocator alloc = {a_malloc, a_realloc, a_free};
 static long num_allocs = 0;
+static bool alloc_has_failed = false;
 
 void base_set_allocator(struct base_allocator *defs)
 {
 	memcpy(&alloc, defs, sizeof(struct base_allocator));
 }
 
+bool is_allocator_failed(void)
```

Comment:
Having troubles to see where this is being used.

Reply:
It exported and will be used from obs-studio-node HandleCrash function.  
https://github.com/stream-labs/obs-studio-node/pull/530/files#diff-01820d80f096932e8d1785d297c0acfb

Reply:
Got it, looks good to me.